test/integration/merkle-lockup/lt/claim/claim.t.sol

```diff
@@ -101,7 +109,69 @@ contract Claim_Integration_Test is MerkleLockup_Integration_Test {
         _;
     }
 
-    function test_Claim() external givenCampaignNotExpired givenNotClaimed givenIncludedInMerkleTree {
+    // Needed this variables in storage due to how the imported libaries work.
+    uint256[] public leaves = new uint256[](4); // same number of recipients as in the defaults
+
+    function test_Claim_TrancheAmountCalculationRoundingError()
+        external
+        givenCampaignNotExpired
+        givenNotClaimed
+        givenIncludedInMerkleTree
+    {
+        // Declare an amount that will cause a rounding error.
+        uint128 claimAmount = 340_282_366_920_938_463_463_374_607_431_768_211_453;
+        uint256 aggregateAmount = defaults.CLAIM_AMOUNT() * 3 + uint256(claimAmount);
+
+        // Compute the Merkle tree.
+        leaves = defaults.getLeaves();
+        leaves[0] = MerkleBuilder.computeLeaf(defaults.INDEX1(), users.recipient1, claimAmount);
+        MerkleBuilder.sortLeaves(leaves);
+        bytes32 merkleRoot = getRoot(leaves.toBytes32());
+
+        // Compute the Merkle proof.
+        uint256 leaf = MerkleBuilder.computeLeaf(defaults.INDEX1(), users.recipient1, claimAmount);
```

Comment:
You can also define this `leaf` above the line in which you substitute `leaves[0]` (L127). Then you can substitute using `leaves[0] = leaf`. This avoids computing leaf two times.

Reply:
we can even remove this line and use `leaves[0]`